website/src/layouts/BaseLayout.astro

```diff
@@ -72,7 +72,7 @@ const lastTimeBannerWasClosed = Astro.cookies.get('lastTimeBannerWasClosed')?.va
                             <OrganismSelector implicitOrganism={implicitOrganism} />
                         </div>
                     </div>
-                    <Navigation implicitOrganism={implicitOrganism} />
+                    <Navigation implicitOrganism={implicitOrganism} gitHubMainUrl={gitHubMainUrl} />
```

Comment:
We could just pass down the whole config instead of unpacking in each component, this reduces number of changes to add extra config items in the future. But not necessary to do now, just a thought when seeing this PR

Reply:
Yeah - sometimes on the website this isn't desirable as it can pass heavy stuff to the client side that doesn't need to be passed. Oh and actually maybe that does apply here to the sandwich menu (but I can't remember the details, maybe its not too heavy). I'll leave it for now.